docs/guide/database-access.md

```diff
@@ -38,22 +38,18 @@ There are references to other core apostrophe collections that can be accessed i
 
 Next, we are invoking a method of the MongoDB collection saying that we want to update one document within the database. You can read about other methods within the [MongoDB Node.js driver docs](https://www.mongodb.com/docs/drivers/node/current/fundamentals/crud/).
 
-We are passing two arguments to the MongoDB method. The first is an object that sets the `_id` property to the `_id` of the piece that the user is requesting. Note the use of the underscore `_` - this is because this is a MongoDB, not user, generated id.
+We are passing two arguments to the MongoDB method. The first is a MongoDB criteria object which ensures our query only matches the document whose `_id` property of the piece that the user is requesting. Note the use of the underscore `_` - this is because this is a MongoDB, not user, generated id.
```

Comment:
"document whose `_id` property of the piece that the user is requesting" -> "with the desired _id"

Reply:
This isn't an _id generated by mongodb per se, so that language isn't quite right. I'd just say "MongoDB objects always have an _id property, a unique identifier that can always be used to find that specific document."

In apostrophe's case _id is a string generated by `cuid()` but they probably don't need to worry about that since they won't be changing it and we don't recommend ever inserting a *new* document using the raw mongo APIs.

Reply:
Changed